lib/stack_master/commands/helper.rb

```diff
@@ -0,0 +1,21 @@
+require 'active_support/concern'
+
+module StackMaster
+  module Commands
+    module Helper
+      extend ActiveSupport::Concern
+
+      included do
```

Comment:
You've already got `active_support` in the Gemfile, I figured that made it fair game.  :-P